mateweather/mateweather-dialog.c

```diff
@@ -502,6 +502,45 @@ static PangoFontDescription* get_system_monospace_font(void)
     return desc;
 }
 
+static void
+override_widget_font (GtkWidget            *widget,
+		      PangoFontDescription *font)
```

Comment:
Indentation...

Reply:
see https://github.com/mate-desktop/mate-applets/blob/weather_override_font/mateweather/mateweather-dialog.c#L505

Reply:
Ah, there is tab... I always use spaces to align function parameters.